### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by johannschopplich**
This pull request fixes some typos in the readme and improves its structure and thus readability (imho).

Sorry, missed the rebase. Base of this PR is https://github.com/rasteiner/k3-pagesdisplay-section/pull/7.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     }
   ],
   "require": {
-    "getkirby/composer-installer": "^1.1"
+    "getkirby/composer-installer": "^2.1"
```
Comment (rasteiner) [inline composer.json]: The last version I see in packagist is 1.2.1. Where does the 2.1 come from? Did you mean 
```
"getkirby/composer-installer": "^1.2"
```
?
Or did I miss something?

File: src/PagesDisplaySection.php [hunk 2]
```diff
@@ -1,61 +1,73 @@
...
+                    unset($pages->data[$id]);
+                    continue;
+                }
+            }
+
+            // Sort pages
+            if ($this->sortBy) {
+                $pages = $pages->sort(...$pages::sortArgs($this->sortBy));
+            }
+
```
Comment (rasteiner) [inline src/PagesDisplaySection.php:55]: Could you add the "flip" option, since we're already here? 🙏

**Review by rasteiner — COMMENTED** [review]
Looks mostly good to me, thanks :)

Could you please take a look at the comments?

File: composer.json (hunk 1 above)
Comment (johannschopplich) [reply composer.json]: Oops, my bad! I meant `1.2`. Fixed in the other pull request. 🙂

File: src/PagesDisplaySection.php (hunk 2 above)
Comment (johannschopplich) [reply src/PagesDisplaySection.php:55]: Good point. Sure! Done. (https://github.com/rasteiner/k3-pagesdisplay-section/pull/7)

**Pull request merged**